src/Session.php

```diff
@@ -934,10 +934,37 @@ public static function checkLoginUser()
         if (!isset($_SESSION["glpiname"])) {
            // Gestion timeout session
             self::redirectIfNotLoggedIn();
-            Html::displayRightError();
+            Html::displayRightError("User has no valid session but seems to be logged in");
         }
     }
 
+    /**
+     * Get the name of the right.
+     *
+     * This only works for well-known rights.
+     * @param int $right The right
+     * @return string The right name
+     * @internal No backwards compatibility promise. Use in core only.
```

Comment:
Note that this function is marked as internal to indicate that it has no BC promise.
I imagine this can be moved into the PermissionManager class after that PR is merged for 10.1.